homeassistant/components/devolo_home_network/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      This integration does not have an options flow.
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: done
+  test-coverage: done
```

Comment:
I am not sure what `test_binary_sensor_setup` is testing, if the binary sensor platform is not setup this test would still pass

Reply:
It tests, if the devices that are enabled by default are setup and those disabled by default are not setup. So if the platform is not setup at all, the assert in line 37 should fail, shouldn't it?

Reply:
Nope, since it if it can't be found it will return `None` by default, and that is what we are testing

Reply:
Oh, now I see it. I was looking at the wrong platform. The binary sensor platform only has one entity and that is disabled by default. So testing the initial setup as I do in the other platforms does not make sence. If I'd remove that test completely, the test coverage would not change because of `test_update_attached_to_router` but the tests would not notice if that default changes. Is there any other best practice to follow in that case?